ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/EVPN_MULTICAST_TESTS.yml

```diff
@@ -98,8 +98,8 @@ tenants:
       257:
         name: "MULTICAST_ENABLED_257"
         tags: ['test_l2']
-      4096:
-        name: "MULTICAST_ENABLED_4096"
+      4092:
+        name: "MULTICAST_ENABLED_4092"
```

Comment:
Changing vlan number to avoid invalid config - which again triggers schema validation errors.